include/ofi_hmem.h

```diff
@@ -134,6 +134,7 @@ int cuda_dev_unregister(uint64_t handle);
 int cuda_get_handle(void *dev_buf, void **handle);
 int cuda_open_handle(void **handle, uint64_t device, void **ipc_ptr);
 int cuda_close_handle(void *ipc_ptr);
+int cuda_get_base_addr(const void *ptr, void **base, size_t *size);
```

Comment:
get base addr isn't really what this does. It's more like, get address range

Reply:
@wckzhang The function name follows the definition of `struct fi_hmem_ops`, which was extended sometime ago from getting base_addr to getting base_addr plus size. Renaming can be considered in the future (I don't see a pressing reason to do so, though), but for this PR the function name is consistent with rest of libfabric.

Reply:
Yes, this is an existing API and I just followed it.